src/bcf-toolkit/Builder/Bcf21/BimSnippetBuilder.cs

```diff
@@ -36,7 +38,7 @@ public class BimSnippetBuilder :
     return this;
   }
 
-  public IBimSnippet Build() {
+  public BimSnippet Build() {
```

Comment:
Yes, I think it did not make any sense to use interface as return type, because the concrete builder must know which type is built. So for the user it is inconvenient to cast after build, plus possibility of error or wrong use.